docs/hub/api.md

```diff
@@ -224,4 +230,100 @@ Payload:
 headers = { "authorization" :  "Bearer $token" }
 ```
 
-This is equivalent to `huggingface_hub.whoami()`.
\ No newline at end of file
+This is equivalent to `huggingface_hub.whoami()`.
```

Comment:
It would be better but auto-generated links won't work here as it's not in the same docs (hub-docs vs huggingface_hub docs). So for now I suggest we keep it like this.

(same below)